claim-check-pattern/call-usage-app/src/main/java/com/iluwatar/claimcheckpattern/utility/MessageHandlerUtility.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * The MIT License
+ * Copyright © 2014-2021 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.claimcheckpattern.utility;
+
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.serializer.TypeReference;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.iluwatar.claimcheckpattern.domain.Message;
+import com.iluwatar.claimcheckpattern.domain.MessageReference;
+import java.util.logging.Logger;
+
+/**
+ * This class read and drop message from Azure blob storage.
+ * @param <T> represents UsageDetail or UsageCostDetail
+ */
+public class MessageHandlerUtility<T> {
+
+  private BlobServiceClient blobServiceClient;
+
+  /**
+   * Parameterized constructor.
+   * @param blobServiceClient represents BlobServiceClient
+   */
+  public MessageHandlerUtility(BlobServiceClient blobServiceClient) {
+    this.blobServiceClient = blobServiceClient;
+  }
+
+  /**
+   * Default constructor.
+   */
+  public MessageHandlerUtility() {
+    // Create a BlobServiceClient object which will be used to create a container
+    // client
+    this.blobServiceClient = new BlobServiceClientBuilder()
+        .connectionString(System.getenv("BlobStorageConnectionString")).buildClient();
+
+  }
+
+  /**
+   * Read message from blob storage.
+   * @param messageReference represents MessageReference
+   * @param logger represents Logger
+   * @return Message
+   */
+  public Message<T> readFromPersistantStorage(MessageReference messageReference, Logger logger) {
+    Message<T> message = null;
+    try {
+
+      // Get container name from message reference
+      String containerName = messageReference.getDataLocation();
```

Comment:
You can use Java 11 local variable type inference (keyword `var`) in here and many other places

Reply:
@iluwatar uses `var` wherever possible for local variables.